res/values/colors.xml

```diff
@@ -398,4 +398,32 @@
     <!-- Overwrite external chips colors -->
     <color name="chips_dropdown_text_default_dark">#9e9e9e</color>
     <color name="chips_dropdown_text_highlighted_dark">@android:color/holo_blue_light</color>
+
+    <!-- default calendar/event color palette -->
+    <array name="defaultCalendarColors">
```

Comment:
Would it make sense to unify these with those from the primary color selector here: https://github.com/Etar-Group/Etar-Calendar/blob/master/res/values/colors.xml#L22 ?

Reply:
Hmm, not sure. My palette is used for offline calendar+events and has been borrowed from the default palette of a big advertisement company.
The colors in https://github.com/Etar-Group/Etar-Calendar/blob/master/res/values/colors.xml#L22 are used for Etar's theme.

Reply:
OK, we can still rethink this when we move to `ColorPickerDialogX` at some point.